keyboards/checkerboards/snop60/readme.md

```diff
@@ -0,0 +1,16 @@
+# SNOP60 
+
+60% Universal PCB feauturing Tsangan bottom row, 2x3u bar support, backlighting, RGB underglow, rotary encoder support, and two flex cutouts. 
+
+- Keyboard Maintainer: [Nasp](https://github.com/npspears)
+- Hardware Supported: SNOP60 PCB
+- Hardware Availability: www.checkerboards.xyz
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
It's a couple lines below this. Do I need to add all three points?